### ISSUE TITLE
[BUG] kirby-item with kirby-toggle will lose screen reader focus when clicked

### ISSUE THREAD

**Issue by sunejohansson** [kirbydesign/designsystem#3957]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When using voice over as a screen reader, when clicking on a toggle that is inside an item, the screen reader can lose focus and the focus will be reset to the browser window instead of the web page. 

https://github.com/user-attachments/assets/d09fc16d-af91-4ddc-8e0b-743152637303

This video shows how the focus is lost from the toggle/item after a click. 

## Describe how to reproduce the bug

The bug can be reproduced by having a `kirby-item` with a `kirby-label` and a `kirby-toggle` inside. When navigating with voice over, it is possible to select two "invisible" elements after highlighting the toggle itself. The two invisible elements can be clicked and will trigger the click event on the toggle, but will then completely lose focus. It might not happen every time. 

See this Stackblitz: https://stackblitz.com/edit/stackblitz-starters-4f9av58g?file=src%2Fmain.ts

1. Use voice over in Chrome
2. Navigate to the item with the toggle. Go past the toggle and see that two small invisible elements will be highlighted. 
3. Click one of the highlighted elements. 
4. Focus will often be lost after the click. 

## Which Kirby version was used?
`10.5.1`

## What was the expected behavior?
That no invisible elements could be focused with the screen reader and that the screen reader's focus would not be lost when clicking the toggle. 

## Please complete the following information:
- OS: MacOS
- Browser Chrome and Safari
- Browser version Chrome 136.0, Safari 18.5

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
